packages/playground/src/portal/dao_view.vue

```diff
@@ -0,0 +1,338 @@
+<template>
+  <v-container fluid>
+    <v-card color="#1AA18F" class="my-3 pa-3 text-center">
+      <v-icon width="26">mdi-note-check-outline</v-icon>
+      <h2>DAO</h2>
+    </v-card>
+    <div class="d-flex my-12 align-center justify-center" v-if="loadingProposals">
+      <v-progress-circular :size="70" :width="7" indeterminate color="primary"></v-progress-circular>
+    </div>
+
+    <div v-else-if="proposals?.active?.length == 0 && proposals?.inactive?.length == 0">
+      <v-card class="my-3 pa-3 d-flex justify-center">
+        <h3>No proposals at this time</h3>
+      </v-card>
+    </div>
+    <div v-else>
+      <v-card>
+        <h4 class="d-flex justify-center pa-4">
+          User, you can now vote on proposals!
```

Comment:
Can we change the phrasing here?

Reply:
It is the same as given in figma [design ](https://www.figma.com/proto/KAQ6497YfczErqfh1Bz62l/Dashboard2?type=design&node-id=54-2667&t=zNC6x0dPbosoH4bW-0&scaling=scale-down&page-id=0%3A1&starting-point-node-id=1%3A1248)